package.json

```diff
@@ -12,6 +12,9 @@
   "bin": {
     "versionist": "./bin/cli.js"
   },
+  "engines": {
+    "node": ">=20"
```

Comment:
balena-semver requires node 16, but that's already EOL so I decided to go to 20.
Let me know if we need to lower in case it makes it harder to bump versionist in other dependent packages.

Reply:
Technically it's not even a dependency of [balena-versionist](https://github.com/product-os/balena-versionist/blob/master/package.json) right now, so it should be fine?